### PR TITLE
Added VCenterPropertiesCollector & Updated VCenterStatsCollector

### PR DESCRIPTION
**Pull request by sunnyrarora**
Requesting a review -Thanks

### CODE REVIEW

File: collectors/VCenterPropertiesCollector.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    def do_metrics(self, target, g, i):
+        token = self.get_target_tokens()
+        token = token[target]
+        if not token:
+            print("skipping " + target + " in , no token")
+
+        for vc in self.get_vcenters():
+            uuid = self.vcenters[vc]['uuid']
+            target_vc = self.vcenters[vc]['name']
+            if 'info_metrics' in self.property_yaml[self.name]:
```
Comment (viennaa) [inline collectors/VCenterPropertiesCollector.py:49]: Please add all type of metrics here:
* info
* number
* enum

The collector should be able to handle all our expected formats

File: collectors/VCenterStatsCollector.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                continue
+
+            try:
+                metric_value = int(values)
+                g.add_metric(labels=[target_vc, statkey_label], value=metric_value)
+
+            except (ValueError, TypeError):
+                info = values
+                metric_value = 0
+                g.add_metric(labels=[target_vc, statkey_label + ":" + info], value=metric_value)
```
Comment (viennaa) [inline collectors/VCenterStatsCollector.py:60]: Why do you need a try catch here? Please add it straight.

File: collectors/VCenterPropertiesCollector.py (hunk 1 above)
Comment (sunnyrarora) [reply collectors/VCenterPropertiesCollector.py:49]: my concern here is - we are already using `summery|version` and `summary|vcfullname` in the collector as for - info_metrics.
For other two metrics like number and enum which Propkey i should pickup ? Only availability as in this picture. And also will that be required ? -Thanks
image

File: collectors/VCenterStatsCollector.py (hunk 2 above)
Comment (sunnyrarora) [reply collectors/VCenterStatsCollector.py:60]: Will work on this to resolve it. -Thanks

**Comment by sunnyrarora** [comment]
Updated the requested changes and removed the try-catch block -Thanks

**Pull request merged**